### PR TITLE
Added the attribute for changing the speed of climbing the ladder

### PR DESCRIPTION
**Pull request by PawBud**
This is a fix for the issue: _#1563_
As suggested by @4ian, I am working on  the JS extension of the attribute

### CODE REVIEW

**Comment by 4ian** [comment]
Your changes in the C++ side are incomplete, you also need to update the header.
This means that GDevelop.js was probably not compiled properly.

When doing changes, you need to test them - it's not just about code :) Read how to compile GDevelop.js, there is a README about it.

**Comment by PawBud** [comment]
yea I realized that

**Comment by PawBud** [comment]
@4ian i have made the required changes though I haven't tested them properly so I would be highly obliged if u may review my code

**Comment by 4ian** [comment]
Can you test them? :)

**Comment by PawBud** [comment]
I will be happy too :).

**Comment by PawBud** [comment]
I think I shud remove those spaces caused by prettier, right @4ian?

**Comment by 4ian** [comment]
Yes. Though they are not caused by prettier, you probably used auto formatting in VSCode without having Prettier installed.

**Comment by PawBud** [comment]
@4ian yea, I just noticed that. I will revert them back and then force push :)

File: Extensions/PlatformBehavior/PlatformerObjectRuntimeBehavior.h [hunk 1]
```diff
@@ -33,6 +33,7 @@ class GD_EXTENSION_API PlatformerObjectRuntimeBehavior
 
   double GetGravity() const { return gravity; };
   double GetMaxFallingSpeed() const { return maxFallingSpeed; };
+  double GetLadderClimbingSpeed() const {return ladderClimbingSpeed};
```
Comment (4ian) [inline Extensions/PlatformBehavior/PlatformerObjectRuntimeBehavior.h]: Be consistent in the spacings

Comment (PawBud) [reply Extensions/PlatformBehavior/PlatformerObjectRuntimeBehavior.h]: Yup, I am trying to imitate the spaces as they have been done in other files.

Comment (PawBud) [reply Extensions/PlatformBehavior/PlatformerObjectRuntimeBehavior.h]: Ohh! I just noticed the `{return.....}`. Thanks for that

**Pull request closed without merging**